### PR TITLE
fix test after unshare table is not available

### PR DESCRIPTION
**Pull request by romanbracinik**
Po dlhsiu dobu sme nic nepridavali tak fixujem test linkovania ktory zacal failovat.

### CODE REVIEW

File: tests/functional/UseCase/Bucket/ShareLinkBucketTest.php [hunk 1]
```diff
@@ -347,14 +354,15 @@ public function testShareUnshareLinkedBucket(): void
...
         $targetDataset = $targetProjectBqClient->dataset($linkedBucketSchemaName);
         $this->assertTrue($targetDataset->exists());
         $testTableBefore = $targetDataset->table(self::TESTTABLE_AFTER_NAME);
-        $this->assertTrue($testTableBefore->exists());
+
+        // after unshare the table is not available
+        // in connection you can't just unshare a bucket that is lined up first so this is an edge case
+        // handled in connection
+        $this->expectException(BadRequestException::class);
+        $testTableBefore->exists();
```
Comment (romanbracinik) [inline tests/functional/UseCase/Bucket/ShareLinkBucketTest.php:366]: Neviem ci sa to stalo tim ze sa nam zmenili opravnenia alebo je to nejaka zmena v BQ nenasiel som nic ktomu. Kazdopadne nam to asi nevadi ze to teraz funguje takto

**Comment by romanbracinik** [comment]
build presiel tu https://github.com/keboola/php-storage-driver-bigquery/actions/runs/5788101603

**Review by zajca — APPROVED** [review]
